### DISCUSSION TITLE
Ability to track changes

### DISCUSSION THREAD

**Discussion by go2santosh** [jgraph/drawio#2975] [Ideas]
I see the .drawio file is saved as a text file in xml format. Currently the entire xml text is saved in a single line. For example, here is how the currently the xml look like:
`<mxfile host="Electron" modified="2022-08-01T..." agent="5.0 (Macintosh; ..." etag="8e0vuj..."  version="19.0.3"  type="device" pages="2"><diagram name="Page-1" id="edf60f1...">3Vpbd6M2E...</diagram><diagram name="Page-2" id="KK8EQf...">5VpbU9s4FP4...</diagram></mxfile>`

I suggest to slightly modify the xml format such that every element appear on new line. After inserting newline after every elements the xml would look like:
`<mxfile host="Electron" modified="2022-08-01T..." agent="5.0 (Macintosh; ..." etag="8e0vuj..."  version="19.0.3"  type="device" pages="2">`
`<diagram name="Page-1" id="edf60f1...">3Vpbd6M2E...</diagram>`
`<diagram name="Page-2" id="KK8EQf...">5VpbU9s4FP4...</diagram>`
`</mxfile>`

This would be helpful for me as usually my drawio files has many pages and whenever I make change in any single page then my code repo file compare feature would clearly show me exactly which page was changed. 

Please enlighten me in case if there is already any better alternate way to achieve similar track change.

**Comment by davidjgraph** [comment]
File->Properties, untick compressed.

**Reply by go2santosh** [reply]
Thanks. Perfect!
